src/app/login/page.tsx

```
@@ -0,0 +1,97 @@
+          <span className="text-lime-500">Welcome!</span> Please sign in to
+          continue
+        </div>
+
+        {/* Social Login */}
+        <div className="space-y-4">
+          <div className="grid grid-cols-2 gap-4">
+            <Button
+              variant="outline"
+              className="w-full border-lime-500 text-lime-500 hover:bg-lime-50"
+            >
+              <svg viewBox="0 0 24 24" className="mr-2 h-4 w-4">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)